frontend/packages/metal3-plugin/src/components/baremetal-hosts/BareMetalHostStatus.tsx

```diff
@@ -48,10 +53,35 @@ const BareMetalHostStatus: React.FC<StatusProps> = ({ status, title, ...props })
       return <AddDiscoveredHostButton host={props.host} />;
     case [NODE_STATUS_STARTING_MAINTENANCE, NODE_STATUS_UNDER_MAINTENANCE].includes(status):
       return <MaintenancePopover title={statusTitle} maintenance={props.maintenance} />;
+    case status === HOST_STATUS_INSPECTING:
+      return (
+        <ProgressStatus title={statusTitle}>
+          The hardware details for the host are being collected by an agent. This operation will
+          take a while. The host will become available when finished.
+        </ProgressStatus>
+      );
+    case status === HOST_STATUS_PROVISIONING:
+      return (
+        <ProgressStatus title={statusTitle}>
+          An image is being written to the host&apos;s disk(s).
+        </ProgressStatus>
+      );
+    case status === HOST_STATUS_DEPROVISIONING:
+      return (
+        <ProgressStatus title={statusTitle}>
+          The image is being wiped from the host&apos;s disk(s).
```

Comment:
```suggestion
          The image is being wiped from the host&apos;s disk(s). This may take a while.
```

I assume this could take a while too depending on how much there is to wipe, so we may as well set expectations low.